### PR TITLE
Add Oryx Pro 9

### PR DESCRIPTION
**Pull request by crawfxrd**
- system76/coreboot#117
- system76/ec#287

### CODE REVIEW

**Comment by leviport** [comment]
Is this ready for QA to review?

**Comment by XV-02** [comment]
So, having flashed it three times to make sure it wasn't a bad-flash, I'm running into a number of problems: 

- The function keys are not working. That is, features like the volume up/down, touchpad lock, screen blanking, &c.

- Suspend/Resume doesn't work in hybrid, integrated, or nvidia modes.
-- In hybrid mode, attempting to resume fails to deliver a log in prompt, and results in a black screen with a solid cursor
-- In integrated, a login prompt displays, but once logged in, the same behaviour as in hybrid mode manifests
-- In nvidia mode, the session returns with the file-system in read-only mode (and the session broken to the extent that the _only way_ to shut the machine down was the power button, as none of the traditional bash commands (`shutdown now`, `poweroff`, `reboot`) were functional.

- Keyboard Configurator doesn't detect the Oryp9 keyboard, though System76-Driver does report correctly, and the "firmware" and "about" settings pages do as well

- Track Pad has been non-responsive on boot a number of times. Graphics mode doesn't appear to matter, even though I saw it a bit more in integrated mode than the others. I think that's more chance than indicative of a root cause. When non-functional, a reboot will usually resolve the issue. Additionally, when non-functional, the trackpad is not reported when running `xinput list`. The reported touchpad for the lab unit is the `ELAN0412:00 04F3:316F`

**Comment by XV-02** [comment]
As a follow on: The suspend issue seems to instance immediately after the laptop hits suspend. Two attached logs show the Oryp9 15" reach suspend before the log can no longer update due to write permission issues. The third log is of a successful suspend on the Oryp9 17"
[SuspendLog1.txt](https://github.com/system76/firmware-open/files/9074647/SuspendLog1.txt) (should be at log level 7)
[SuspendLog2.txt](https://github.com/system76/firmware-open/files/9074648/SuspendLog2.txt) 
[SuccessfulSuspend.txt](https://github.com/system76/firmware-open/files/9074649/SuccessfulSuspend.txt)

**Comment by XV-02** [comment]
Closing the lid to suspend doesn't seem to work on either unit.

**Comment by XV-02** [comment]
Okay, so, discard previous noted problems. With the EC confirmed good flash, and good firmware, most are resolved. So far, new issues include:

- On resume, machine reports `EXT4-fs error (device nvme0n1p3): __ext4_find_entry:1635: inode #19660801: comm gmain: reading directory lblock 0`
- If resume resolves to a usable session, file system reports are read-only.

**Comment by crawfxrd** [comment]
What drives are you using?

**Comment by XV-02** [comment]
A single Western Digital blue 500Gb drive.

**Comment by XV-02** [comment]
Oryp9 Doesn't reconnect to external monitor when set up in a closed-lid manner with a thunderbolt dock.

**Comment by crawfxrd** [comment]
Does this work on the other ADL boards? I don't have a dock to test with.

**Comment by XV-02** [comment]
It does. The Gaze17, it works correctly on.

**Comment by crawfxrd** [comment]
- Updated submodules after merging PRs
- Updated changelog
- Updated list of supported models

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,12 @@ Changes are identified by the date of the released firmware including them. If
 you are running System76 Open Firmware, opening the boot menu will show this
 date followed by an underscore and a short git revision.
 
-## 2022-07-05
+## 2022-07-12
 
-- lemp11: Fix power off under load while on battery power
+- oryp9: Release of open firmware with System76 EC
+- darp8: Fixed power off under load while on battery power
```
Comment (crawfxrd) [inline CHANGELOG.md:10]: Did we hold shipping darp8 for this fix?

Comment (leviport) [reply CHANGELOG.md:10]: Pretty sure we didn't, so this should be a darp8 update

**Review by XV-02 — APPROVED** [review]
Firmware passed certification https://github.com/system76/certification/blob/master/laptops/oryp9/2022-07-12_3080Ti%263070Ti_Pop2204_5b070b1.md

**Review by 13r0ck — APPROVED** [review]

**Pull request merged**